packages/gravity-ui-web/src/sass/05-components/03-organisms/_hero.scss

```diff
@@ -1,28 +1,33 @@
 .grav-c-hero {
   position: relative;
-  padding: ($grav-sp-xl + $grav-sp-l) 0 $grav-sp-xl 0;
   text-align: center;
 
-  .grav-c-pullout,
-  p {
-    margin-top: 0;
-  }
-
-  p {
-    margin-bottom: $grav-sp-l;
-  }
-
-  > canvas {
+  .grav-c-hero__banner {
     width: 100%;
-    max-width: initial;
-    height: 100%;
+    height: 420px;
     position: absolute;
-    top: 0;
-    left: 0;
-    margin: 0;
+    background: url('/images/hero.jpg') center;
+    background-size: cover;
+    z-index: 0;
   }
 
-  > :not(canvas) {
+  .grav-c-hero__container {
```

Comment:
I think we need to make some changes to the _horiztonal_ layout of this too. Currently, there are some issues:

* The container's width matches the page's content area (since you're using `grav-o-container`), but in the designs it's inset a bit: <img width="583" alt="image" src="https://user-images.githubusercontent.com/16718916/57852427-1081dc80-77db-11e9-8b5e-f62aa7e57d61.png">
* On narrow viewports, the generous padding and large font sizes don't work well: <img width="855" alt="image" src="https://user-images.githubusercontent.com/16718916/57857637-74120700-77e7-11e9-925c-4a2879abbb2f.png"> 

What I propse is that we:
* Make the container appear inset from the edge of the viewport by a small amount (perhaps `$grav-sp-m`) on small viewports, and then exapnd to a `max-width` that is inset from the content area's max-wdith by a larger amount (as per the design). Using horiztonal `auto` margins, we can ensure it remains centered in all cases. I've mocked it up in my browser's DevTools (I added a red border simply to visualise the edge of the container better): 
<img width="419" alt="image" src="https://user-images.githubusercontent.com/16718916/57858623-4d54d000-77e9-11e9-8812-116611b0f7b0.png"> <img width="857" alt="image" src="https://user-images.githubusercontent.com/16718916/57858682-652c5400-77e9-11e9-8d5a-1656a22270dc.png"> <img width="1614" alt="image" src="https://user-images.githubusercontent.com/16718916/57858715-76756080-77e9-11e9-9775-065c4e160b18.png">
* To this we just need 3 rules:
    * `margin: 0 auto;` to keep it centered within its parent
    * `width: calc(100% - #{ 2 * $grav-sp-m });` so that its width is the parent's width minus 2 * our small inset
    * `max-width: calc(#{ $largest-breakpoint-width } - #{ 2 * $hero-container-large-inset });` so that it stops expanding once it hits the width of the largest breakpoint minus whatever the larger inset needs to be.
* Next, we shoud set a smaller padding on `.grav-c-hero__container` - I suggest `$grav-sp-m` - which will work well on smaller viewports. Note that we should only set this along left, right and bottom sides. The top padding needs to be bigger - see comment below. To keep the content within further away from the content's edges on larger viewports, we can again use `max-width`. So, something like:

```sass
.grav-c-hero__container {

  > * {
     // Can't use margin shorthand here because we want top &
     // bottom to inherit their defaults
     margin-left: auto;
     margin-right: auto;

     max-width: $hero-container-content-max-width; // Pick a suitable value in REM based on the design
  }
}
```

Reply:
I wasn't quite able to get what you have here working. But I came up with an implementation that creates the same end result. I think it works pretty well with fairly minimal code.

Reply:
No worries.

Hmm.. your solution does work - although there is a slight jump with the padding inside the container when you're resizing the viewport past the breakpoint.

I had a go at implementing the change I was proposing and I've got a working solution. I pushed it to the `proposed-hero-tweaks` branch so you can take a look. (I also updated the JSON files so the realistic text and background image appear on the homepage and elsewhere we show dummy content)

If you agree with those changes, you can merge it into this branch.